ingestion/utils.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import time
+from datetime import timedelta
+
+logging.basicConfig()
+logging.getLogger(__name__).setLevel(logging.DEBUG)
+
+
+def st_time(func):
+    """
+    st decorator to calculate the total time of a func
```

Comment:
What's `st` stand for?

Reply:
No idea - modelled after something from SO. Can be changed to something more meaningful.

Reply:
maybe `report_time`?